docs/ref/migration-operations.txt

```diff
@@ -269,7 +269,7 @@ be removed (elided) when :ref:`squashing migrations <migration-squashing>`.
 RunPython
 ---------
 
-.. class:: RunPython(code, reverse_code=None, atomic=True, hints=None, elidable=False)
+.. class:: RunPython(code, reverse_code=None, atomic=None, hints=None, elidable=False)
```

Comment:
I agree the matrix is a bit much. Doesn't the sentence I pointed out need to be revised though?